include/uv.h

```diff
@@ -980,6 +982,20 @@ UV_EXTERN int uv_queue_work(uv_loop_t* loop,
 
 UV_EXTERN int uv_cancel(uv_req_t* req);
 
+typedef void (*uv_queue_stats_cb)(int queued, int idle_threads, void* data);
```

Comment:
I'd like to change this to to `typedef void (*)(uv_queue_stats_t*, int, int)`, that way the struct can be embedded in another data structure a la `container_of()`.

Side question: is the use of `int` (vs. for example `unsigned`) intentional?

Reply:
int is a bit friendlier to users, IMO, but unsigned reflects better that it can't be negative. went back and forth on this a bit in my head when writing it. Now its unsigned.